inference-engine/tests/functional/plugin/myriad/subgraph_tests/dsr_strided_slice.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "dsr_tests_common.hpp"
+
+#include <functional_test_utils/layer_test_utils.hpp>
+#include <ngraph_functions/builders.hpp>
+#include <vpu/ngraph/operations/dynamic_shape_resolver.hpp>
+
+namespace {
+
+using DataType = ngraph::element::Type_t;
+using DataDims = ngraph::Shape;
+
+using namespace LayerTestsUtils::vpu;
+
+struct StridedSliceParams {
+    DataShapeWithUpperBound inputShape;
+    std::vector<int64_t> begin;
+    std::vector<int64_t> end;
+    std::vector<int64_t> strides;
+    std::vector<int64_t> beginMask;
+    std::vector<int64_t> endMask;
+    std::vector<int64_t> newAxisMask;
+    std::vector<int64_t> shrinkAxisMask;
+    std::vector<int64_t> ellipsisMask;
+};
+
+using Parameters = std::tuple<
+        StridedSliceParams,
+        DataType,   // Net precision
+        std::string // Device name
+>;
+
+class DSR_StridedSlice : public testing::WithParamInterface<Parameters>,
+                         public DSR_TestsCommon {
+protected:
+    std::shared_ptr<ngraph::Node> createTestedOp() override {
+        StridedSliceParams params;
+        DataType netPrecision;
+        std::tie(params, netPrecision, targetDevice) = this->GetParam();
+
+        const auto input = createInputSubgraphWithDSR(netPrecision, params.inputShape);
+        auto ss = ngraph::builder::makeStridedSlice(input, params.begin, params.end, params.strides, netPrecision, params.beginMask,
+                                                    params.endMask, params.newAxisMask, params.shrinkAxisMask, params.ellipsisMask);
+
+        return ss;
```

Comment:
A more readable name would be better than two letters.

Reply:
Done.